partner_compassion/views/partner_compassion_view.xml

```diff
@@ -96,7 +96,8 @@
 
             <!-- Add the church members list in a church partner -->
             <xpath expr="//page[2]" position="after">
-                <page string="Members" attrs="{'invisible': [('is_church','=',False), ('member_ids', '=', [])]}" autofocus="autofocus">
+                <page string="Members" attrs="{'invisible': ['|',('is_church','=',False), ('member_ids', '=', [])]}" autofocus="autofocus">Member(
+                    <field name="church_member_count"/>)
```

Comment:
This is not the way of displaying info. Instead of directly writing strings `Members()` outside of fields you should rather add
```
<label for="church_member_count"/> -- this will display the label for the field
<field name="church_member_count"/>
```